bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,1030 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="ehub-wloadconsq">
+		<item-type>Number:Energy</item-type>
+		<label>EHUB, WLoadConsq</label>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-group-type id="ehub-wloadconsq">
+		<label>EHUB, WLoadConsq</label>
```

Comment:
```suggestion
		<label>WLoadConsq</label>
```

The thing structure has already improved much, thanks. When looking at these channels i see some more room for improvement. 
- all channels labels have a prefix that corrosponds to the thing it belongs to, making the prefix redundant. Please remove
- the term it self corrosponds to the api field name making it a technical name. I prefer to change this to a label that indicates the functional use
- there are still many channel types that are identical except for their label, these can be re-used.

hope this makes sense as this applies to all channels in this file.

Reply:
Hi,
done changes in channels.xml and ferroamp.xml, Please have a look!
This time it even works, it just not look OK :-)

We have had conversion with Ferroamp about the information in the External API specification but this is not a high level issue for them so this is the closest we are. I liked to have better Labels but I don't know more than this, Sorry for that.